newsroom/news_api/news/atom/views.py

```diff
@@ -64,7 +64,7 @@ def _format_update_date(date):
     # TODO-ASYNC: replace once the service is ready
     response = await get_internal("news/search")
     company = get_company_or_none_from_request(None)
-    products = get_products_by_company(company.to_dict(context={"use_objectid": True}) if company else None)
+    products = await get_products_by_company(company.to_dict(context={"use_objectid": True}) if company else None)
```

Comment:
I'm now wondering if we should default `use_objectid` to True, so we will always get ObjectId instances where required. I will look into that